linkcheck/htmlutil/linkparse.py

```diff
@@ -86,13 +87,13 @@
 _quoted_pat = r"('[^']+'|\"[^\"]+\"|[^\)\s]+)"
 css_url_re = re.compile(r"url\(\s*(?P<url>%s)\s*\)" % _quoted_pat)
 swf_url_re = re.compile("(?i)%s" % urlutil.safe_url_pattern)
-c_comment_re = re.compile(r"/\*.*?\*/", re.DOTALL)
+c_comment_re = re.compile(b"/\*.*?\*/", re.DOTALL)
```

Comment:
This is going to cause SyntaxWarnings on Python 3.8 because `\*` is not a valid escape in non-raw string literals.

(I sometimes use `[*]` instead of `\*` in regexes to avoid multilevel quoting madness.)

Reply:
This change is actually reverted in a subsequent commit:
https://github.com/linkchecker/linkchecker/pull/40/commits/7c135790ed2f89915c6e39e5b19a4f2e083fe65b
I'll merge that in with this one. N.B. it also adds one other line.